packages/react-positioning/src/usePopper.ts

```diff
@@ -218,6 +219,33 @@ function usePopperOptions(options: PopperOptions, popperOriginalPositionRef: Rea
           enabled: !!arrow,
           options: { element: arrow, padding: arrowPadding },
         },
+
+        /**
+         * Modifies popper offsets to cover the reference rect, but still keep edge alignment
+         */
+        {
+          name: 'coverTarget',
+          enabled: !!coverTarget,
+          phase: 'main',
+          requiresIfExists: ['offset', 'preventOverflow', 'flip'],
+          fn({ state }: PopperJs.ModifierArguments<{}>) {
+            const basePlacement = state.placement.split('-')[0];
```

Comment:
We also use this in `applyMaxSize` modifier, would nice to extract it to a function with proper typings:

```ts
function getBasePlacement(placement: PopperJs.Placement): PopperJs.BasePlacement {
  return state.placement.split('-')[0]
}
```

Reply:
I didn't see that, good idea